framework/src/application.ts

```diff
@@ -54,6 +53,7 @@ import { SidechainInteroperabilityModule } from './modules/interoperability/side
 import { MainchainInteroperabilityModule } from './modules/interoperability/mainchain/module';
 import { SidechainInteroperabilityMethod } from './modules/interoperability/sidechain/method';
 import { MainchainInteroperabilityMethod } from './modules/interoperability/mainchain/method';
```

Comment:
Hi @shuse2 , this is not part of the PR but it's a small fix  😉. Maybe 
```ts
import {
	SidechainInteroperabilityModule,
	MainchainInteroperabilityModule,
	SidechainInteroperabilityMethod,
	MainchainInteroperabilityMethod,
} from './modules/interoperability';
```

Not a show stopper though.